doc/classes/TreeItem.xml

```diff
@@ -119,6 +119,10 @@
 			</return>
 			<description>
 				Returns the TreeItem's first child item or a null object if there is none.
+				To return further children call [method get_next] on the returned object.
+				[codeblock]var child = get_children()
+				while child != null:
+				    child = child.get_next()[/codeblock]
```

Comment:
```suggestion
				To return further children, call [method get_next] on the returned object.
				[codeblock]
				var child = get_children()
				while child != null:
				    child = child.get_next()
				[/codeblock]
```

Reply:
Actually this part was no longer needed as `get_children()` now returns an array.